app/config/passport.js

```diff
@@ -0,0 +1,85 @@
+let connection = require('../models/db.js');
+
+const bcrypt = require('bcrypt');
+const passport = require('passport');
+const passportJWT = require('passport-jwt');
+const JWTStrategy = passportJWT.Strategy;
+const { ExtractJwt } = passportJWT;
+const LocalStrategy = require('passport-local').Strategy;
+
+// MySQL connection 실행
+connection.connect(error=>{
+	if(error) throw error;
+	console.log("Successfully connected to the database. ");
+});
+
+
+const LocalStrategyOption = {
+	usernameField: 'email',
```

Comment:
Codacy has a fix for the issue: [Replace `'email'` with `"email"`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
	usernameField: "email",
```